backend/apps/core/apps.py

```
@@ -0,0 +1,20 @@
+            from django_q.models import Schedule
+            # Create a daily schedule to run the GDPR purge pipeline
+            Schedule.objects.get_or_create(
+                func="apps.core.tasks.purge_expired_soft_deleted_records",
+                defaults={
+                    "schedule_type": Schedule.DAILY,
+                    "repeats": -1, # Infinite repeats
+                }
+            )
+        except Exception:
+            # Table might not be migrated yet or django_q not installed
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)